src/Extension.php

```diff
@@ -42,17 +53,25 @@ public function configure(ArrayNodeDefinition $builder)
         $builder
             ->addDefaultsIfNotSet()
             ->children()
-                ->scalarNode('autoload')
-                    ->defaultValue(true)
+                ->booleanNode('autoload')
+                    ->info('When true, the extension will load the data fixtures for all registered bundles')
+                    ->defaultTrue()
                 ->end()
-                ->variableNode('migrations')
-                    ->defaultNull()
+                ->booleanNode('use_backup')
+                    ->info('When true, the extension will backup the database and restore it when needed')
+                    ->defaultFalse()
```

Comment:
shouldn't we do some smart autodetect here? like `true` for sqlite, mysql and pgsql and false by defaul for all other? or just true by default while not working when not supported?

Reply:
Not sure this is a good idea.
This feature is now enabled by default (I disabled it for my tests).

Reply:
True by default is also a good choice for me